### DISCUSSION TITLE
Ambient light settings

### DISCUSSION THREAD

**Discussion by SimonBugaychuk** [seerge/g-helper#2437] [Q&A] [answered]
Hello @seerge 

Is there a way to use your Ambient light settings in Armory Crate? Or some separate software that will change the settings **only** for the backlight? The default settings in Armory Crate are terrible and I was able to achieve a similar result as in your program by manually adjusting the laptop's backlight zones using Aura Creator. But when I found and used your program I noticed that it works much better in Ghelper. I am used to using the Armory Crate, but would like to know how I can achieve the same lighting while still using the Armory Crate.

Thank you

**Comment by seerge** [comment, marked as answer]
@SimonBugaychuk hello, no, that's not possible. You need to use app or manually export all code / functionality related to Ambient mode somewhere in order to use it separately.
